error_handling.md

```diff
@@ -318,15 +317,13 @@ unknown (not previously defined with `define-opts') option."))
 
 ## Signaling (throwing) conditions
 
-We can use `error`, like we did above. Two ways:
+We can use `error`, like we did above, in two ways:
 
 - `(error "some text")`: signals a condition of type `simple-error`
 - `(error 'my-error :message "We did this and this and it didn't work.")`
 
 Throwing these conditions will enter the interactive debugger,
-where a few options will be presented by default. We can give
```

Comment:
Maybe "use `signal` and allow the user to select a restart?"

Reply:
Agreed